draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -4332,80 +4218,124 @@ RESPONSE:
   Content-Type: application/cdni; ptype=ci-trigger-collection
 
   {
-      "staleresourcetime": 86400,
-      "triggers": [
-          "https://dcdn.example.com/triggers/1"
-      ]
+    "staleresourcetime": 86400,
+    "triggers": [ "https://dcdn.example.com/triggers/1" ]
   }
 ]]></artwork>
    </figure>
    </section>
 
         <section anchor="trigger-with-extensions" title="Extensions with Error Propagation">
```

Comment:
We illustrate location-policy and time-policy here, but not execution-policy.  Is it worth perhaps augmenting this example?